docs/proposals/scheduling/20220629-fine-grained-device-scheduling.md

```diff
@@ -355,6 +341,59 @@ type DeviceList struct {
 }
 ```
 
+##### Compatible
+
+Considering that some users already have many existing GPU Pods in their clusters, it is necessary to ensure that Koordinator GPU Scheduling does not repeatedly allocate the GPU devices held by these GPU Pods. Therefore, koord-scheduler needs to obtain the GPU devices's information held by these existing Pods. These GPU devices are allocated by the kubelet and recorded in the local file `/var/lib/kubelet/device-plugins/kubelet_internal_checkpoint`, so the device reporter will parse the file to obtain the GPU Device ID assigned to each Pod. When parsing, it needs to exclude the Pod that allocates GPU through koord-scheduler, and finally update it to Device CRD in the form of annotation. The corresponding annotation key is `node.koordinator.sh/gpu-checkpoints`, and the annotation value is defined as follows:
```

Comment:
Don't you need to patch the allocation result into existing Pods?

Reply:
Restrict koordlet permissions. Koordlet does not require Pod write permissions, and in some user scenarios, permissions on a single machine are also restricted.